pkg/taskservice/mem_task_storage.go

```diff
@@ -40,6 +41,8 @@ func NewMemTaskStorage() TaskStorage {
 		taskIndexes:     make(map[string]uint64),
 		cronTasks:       make(map[uint64]task.CronTask),
 		cronTaskIndexes: make(map[string]uint64),
+		preUpdateCron:   func() error { return nil },
```

Comment:
Can you add some comment on what is preUpdate?   Who should write this, the taskservice user or taskservice system?    What does it suppose to update?

Reply:
it's a mem task storage implement. Only used to testing, the preUpdateCron used to returns a error for testing